### PR TITLE
add robots instructions for web search crawlers

### PR DESCRIPTION
**Pull request by Tr4pSt3R**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -6,6 +6,8 @@ def not_implemented
 
   get '/' => redirect('/en')
 
+  match '/robots', to: 'sitemap#robots', via: :get
```
Comment (tomas-stefano) [inline config/routes.rb]: You can use get and remove the via option

File: spec/controllers/sitemap_controller_spec.rb [hunk 2]
```diff
@@ -1,4 +1,12 @@
 RSpec.describe SitemapController, type: :controller do
+  describe 'GET robot' do
+    it 'renders robot text file' do
+      get :robots
+
+      expect(response).to be_success
```
Comment (tomas-stefano) [inline spec/controllers/sitemap_controller_spec.rb]: I would change this expectation to render template one. See the description of the spec and the expectation of it.

**Comment by asmega** [comment]
:+1:

**Comment by Tr4pSt3R** [comment]
@asmega I have made a quick change, per @tomas-stefano's recommendation. This PR will need a review again

**Comment by tomas-stefano** [comment]
Cool. Nice work :+1:

**Comment by asmega** [comment]
:+1:

**Pull request merged**